bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) const DEFAULT_FEATURE_PROOF_OF_WORK: bool = true;
 pub(crate) const DEFAULT_WHITE_FLAG_SOLIDIFICATION_TIMEOUT: Duration = Duration::from_secs(2);
 
 /// REST API configuration builder.
-#[derive(Default, Deserialize, PartialEq)]
+#[derive(Default, Deserialize, PartialEq, Eq)]
```

Comment:
Order

Reply:
Hmm yeah I can but overall they are not ordered :/

Reply:
Ok do what you want

Reply:
I've done it all good 👍🏻